backend/src/apiserver/config/sample_config.json

```diff
@@ -5,7 +5,7 @@
     "file": "/samples/core/xgboost_training_cm/xgboost_training_cm.py.yaml"
   },
   {
-    "name": "[Demo] TFX - Taxi Tip Prediction Model Trainer",
+    "name": "[Demo] Pre-configured classification pipeline",
     "description": "[source code](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252Fmaster%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb) [GCP Permission requirements](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/parameterized_tfx_oss#permission). Example pipeline that does classification with model analysis based on a public tax cab dataset.",
```

Comment:
the [source code] link here is not to github. we may update it. it points to Notebook while Xgboost points to github.

Reply:
Very good catch! updated accordingly.